modules/nw-ipfailover-configuration.adoc

```diff
@@ -41,7 +41,42 @@ $ oc adm policy add-scc-to-user privileged -z ipfailover
 ----
 $ oc adm policy add-scc-to-user hostnetwork -z ipfailover
 ----
+
+. {rh-openstack-first} only: Complete the following steps to make the failover VIP addresses reachable on {rh-openstack} ports:
++
+.. Use the {rh-openstack} CLI to find values in the `allowed_address_pairs` parameter of your {rh-openstack} cluster.
++
+[source,terminal]
+----
+$ openstack port show <cluster_name> -c allowed_address_pairs
+----
++
+[source,terminal,subs="attributes,quotes"]
+----
+*Field*                  *Value*
+allowed_address_pairs    ip_address='192.168.0.5', mac_address='fa:16:3e:5d:f0:2d'
+                         ip_address='192.168.0.7', mac_address='fa:16:3e:5d:f0:2d'
+----
++
+.. Add an outputted allowed address pair to the `OPENSHIFT_HA_VIRTUAL_IPS` IP failover environment variable:
```

Comment:
this step will happen after the third one. 'Make the failover VIP addresses reachable on {rh-openstack} ports by entering the following command in the {rh-openstack} CLI: '. Interchange the ii and iii steps

Reply:
:heavy_check_mark: